[CCE] Node attach failed when attaching a prePaid instance

Error messages below reported when I attach a prePaid ECS instance into CCE:
```
2022-09-22T20:09:07.651+0800 [DEBUG] provider.terraform-provider-huaweicloud_v1.40.1: API Response Code: 200: timestamp=2022-09-22T20:09:07.651+0800
2022-09-22T20:09:07.651+0800 [DEBUG] provider.terraform-provider-huaweicloud_v1.40.1: API Response Headers:
Cache-Control: no-cache, no-store, must-revalidate
Cache-Control: no-cache, no-store, must-revalidate
Connection: keep-alive
Content-Type: application/json
Date: Thu, 22 Sep 2022 12:09:07 GMT
Pragma: no-cache
Pragma: no-cache
Server: api-gateway
Set-Cookie: ingress-cfe-cluster-manager-6443=279b515f9e09a03bda2bb83619ba07ab; Path=/; Secure; HttpOnly
Strict-Transport-Security: max-age=31536000; includeSubdomains;
X-Content-Type-Options: nosniff
X-Download-Options: noopen
X-Frame-Options: SAMEORIGIN
X-Request-Id: 27624219d67fddd2f9f95d8feb25abc3
X-Xss-Protection: 1; mode=block;: timestamp=2022-09-22T20:09:07.651+0800
2022-09-22T20:09:07.652+0800 [DEBUG] provider.terraform-provider-huaweicloud_v1.40.1: API Response Body: {
  "apiVersion": "v3",
  "kind": "Node",
  "metadata": {
    "annotations": {
      "kubernetes.io/node-pool.id": "DefaultPool",
      "os.architecture": "amd64"
    },
    "creationTimestamp": "2022-09-22 12:05:47.336298 +0000 UTC",
    "name": "test",
    "uid": "xxxxxx-xxxxx-xxxx-xxxxx-xxxxxxxxxx",
    "updateTimestamp": "2022-09-22 12:08:42.705957 +0000 UTC"
  },
  "spec": {
    "az": "cn-east-3c",
    "billingMode": 1,
    "dataVolumes": [
      {
        "hw:passthrough": true,
        "metadata": {
          "__system__server_id": "xxxxxx-xxxxx-xxxx-xxxxx-xxxxxxxxxx",
          "attached_mode": "rw",
          "hw:passthrough": "true",
          "orderID": "CSXXXXXXXXXXXXXXX",
          "productID": "xxxxx-xxxxxx-x--x",
          "readonly": "False"
        },
        "size": 100,
        "volumetype": "SAS"
      }
    ],
    "ecsGroupId": "xxxxxx-xxxxx-xxxx-xxxxx-xxxxxxxxxx",
    "extendParam": {
      "chargingMode": 1,
      "orderID": "",
      "productID": ""
    },
    "flavor": "c7.2xlarge.4",
    "isStatic": true,
    "login": {
      "sshKey": "xxxxx",
      "userPassword": {}
    },
    "nodeNicSpec": {
      "primaryNic": {
        "subnetId": "xxxxxx-xxxxx-xxxx-xxxxx-xxxxxxxxxx"
      }
    },
    "os": "EulerOS 2.5",
    "publicIP": {
      "eip": {
        "bandwidth": {}
      }
    },
    "rootVolume": {
      "hw:passthrough": true,
      "metadata": {
        "__system__server_id": "xxxxxx-xxxxx-xxxx-xxxxx-xxxxxxxxxx",
        "attached_mode": "rw",
        "billing": "1",
        "hw:passthrough": "true",
        "orderID": "CSXXXXXXXXXXXXXXX",
        "productID": "xxxxx-xxxxxx-x--x",
        "readonly": "False"
      },
      "size": 40,
      "volumetype": "SAS"
    },
    "runtime": {
      "name": "docker"
    }
  },
  "status": {
    "phase": "Active",
    "privateIP": "xxx.xxx.xxx.xxx",
    "serverId": "xxxxxx-xxxxx-xxxx-xxxxx-xxxxxxxxxx"
  }
}: timestamp=2022-09-22T20:09:07.652+0800
2022-09-22T20:09:07.654+0800 [ERROR] provider.terraform-provider-huaweicloud_v1.40.1: setting state: Invalid address to set: []string{"charging_mode"}: timestamp=2022-09-22T20:09:07.654+0800
```

--- a/docs/resources/cce_node_attach.md
+++ b/docs/resources/cce_node_attach.md
@@ -100,6 +100,7 @@ In addition to all arguments above, the following attributes are exported:
 * `runtime` - The runtime of the CCE node.
 * `ecs_group_id` - The Ecs group ID.
 * `subnet_id` - The ID of the subnet to which the NIC belongs.
+* `charging_mode` - The charging mode of the CCE node. Valid values are *prePaid* and *postPaid*.
 
 ## Timeouts
 

--- a/huaweicloud/services/acceptance/cce/resource_huaweicloud_cce_node_attach_test.go
+++ b/huaweicloud/services/acceptance/cce/resource_huaweicloud_cce_node_attach_test.go
@@ -58,6 +58,37 @@ func TestAccCCENodeAttachV3_basic(t *testing.T) {
 	})
 }
 
+func TestAccCCENodeAttachV3_prePaid(t *testing.T) {
+	var node nodes.Nodes
+
+	rName := fmt.Sprintf("tf-acc-test-%s", acctest.RandString(5))
+	resourceName := "huaweicloud_cce_node_attach.test"
+	//clusterName here is used to provide the cluster id to fetch cce node.
+	clusterName := "huaweicloud_cce_cluster.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
+			acceptance.TestAccPreCheckChargingMode(t)
+		},
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      testAccCheckCCENodeV3Destroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCCENodeAttachV3_prePaid(rName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckCCENodeV3Exists(resourceName, clusterName, &node),
+					resource.TestCheckResourceAttr(resourceName, "name", rName),
+					resource.TestCheckResourceAttr(resourceName, "tags.foo", "bar"),
+					resource.TestCheckResourceAttr(resourceName, "tags.key", "value"),
+					resource.TestCheckResourceAttr(resourceName, "os", "EulerOS 2.5"),
+					resource.TestCheckResourceAttr(resourceName, "charging_mode", "prePaid"),
+				),
+			},
+		},
+	})
+}
+
 func testAccCCENodeAttachV3_Base(rName string) string {
 	return fmt.Sprintf(`
 %s
@@ -167,4 +198,76 @@ resource "huaweicloud_cce_node_attach" "test" {
   }
 }
 `, testAccCCENodeAttachV3_Base(rName), rNameUpdate)
+}
+
+func testAccCCENodeAttachV3_prePaidBase(rName string) string {
+	return fmt.Sprintf(`
+%s
+
+data "huaweicloud_availability_zones" "test" {}
+
+data "huaweicloud_images_image" "test" {
+  name        = "EulerOS 2.5 64bit"
+  most_recent = true
+}
+
+resource "huaweicloud_compute_instance" "test" {
+  name                        = "%s"
+  image_id                    = data.huaweicloud_images_image.test.id
+  flavor_id                   = "sn3.large.2"
+  availability_zone           = data.huaweicloud_availability_zones.test.names[0]
+  admin_pass                  = "Test@123"
+  delete_disks_on_termination = true
+
+  charging_mode = "prePaid"
+  period_unit   = "month"
+  period        = 1
+  
+  system_disk_type = "SAS"
+  system_disk_size = 40
+  
+  data_disks {
+	type = "SAS"
+	size = "100"
+  }
+  
+  network {
+	uuid = huaweicloud_vpc_subnet.test.id
+  }
+
+  lifecycle {
+    ignore_changes = [
+      image_id, tags, name
+    ]
+  }
+}
+
+resource "huaweicloud_cce_cluster" "test" {
+  name                   = "%s"
+  cluster_type           = "VirtualMachine"
+  flavor_id              = "cce.s1.small"
+  vpc_id                 = huaweicloud_vpc.test.id
+  subnet_id              = huaweicloud_vpc_subnet.test.id
+  container_network_type = "overlay_l2"
+}
+`, testAccCCEClusterV3_Base(rName), rName, rName)
+}
+
+func testAccCCENodeAttachV3_prePaid(rName string) string {
+	return fmt.Sprintf(`
+%s
+
+resource "huaweicloud_cce_node_attach" "test" {
+  cluster_id = huaweicloud_cce_cluster.test.id
+  server_id  = huaweicloud_compute_instance.test.id
+  password   = "Test@123"
+  os         = "EulerOS 2.5"
+  name       = "%s"
+
+  tags = {
+    foo = "bar"
+    key = "value"
+  }
+}
+`, testAccCCENodeAttachV3_prePaidBase(rName), rName)
 }

--- a/huaweicloud/services/cce/resource_huaweicloud_cce_node_attach.go
+++ b/huaweicloud/services/cce/resource_huaweicloud_cce_node_attach.go
@@ -232,6 +232,10 @@ func ResourceCCENodeAttachV3() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"charging_mode": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
 		},
 	}
 }

